server/test/flag.spec.js

```diff
@@ -0,0 +1,47 @@
+let chai = require('chai');
+let chaiHttp = require('chai-http');
```

Comment:
everything in this file should be part of the article test

Reply:
ok sir. i would move them now